packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -1168,6 +1153,66 @@ export class ThinEngine extends AbstractEngine {
         this._bindUnboundFramebuffer(null);
     }
 
+    /**
+     * Resolves the MSAA textures of the render target into their non-MSAA version
+     * Note that if "texture" is not a MSAA render target, no resolve is performed but mipmaps will still be generated (except if disableGenerateMipMaps is true or if a texture has generateMipMaps equals to false).
+     * @param texture  The render target texture containing the MSAA textures to resolve
+     * @param disableGenerateMipMaps Defines a boolean indicating that mipmaps must not be generated (default: false)
+     */
+    public resolveFramebuffer(texture: RenderTargetWrapper, disableGenerateMipMaps = false): void {

```

Comment:
Can we move this in engine.MSAARenderTarget with all the MSAA thinggies ? or at least the render target ones ?

Reply:
I'm not sure where to move it (`engine.renderTarget.ts` or `engine.multiRender.ts`), as it applies to both single and multi render target case...

Reply:
either it s own module or multirender if possible as I d like to preserve the footprint of ppl already using thinEngine and renderTarget.

Reply:
I did some changes, let me know if it's ok for you.